product-composite-service/pom.xml

```diff
@@ -7,6 +7,7 @@
 		<groupId>com.siriusxi.ms.store</groupId>
 		<artifactId>store-chassis</artifactId>
 		<version>0.0.1-SNAPSHOT</version>
+		<relativePath>../store-chassis</relativePath>
```

Comment:
It seems to work. ;-) What actually happens when you run all the projects together, (mvn -f store-chassis verify) is that Maven can't find the parent in the reactor, because the default value for relativePath is `..` and that it uses the store-chassis pom from the local repository. If you change something in the store-chassis pom and don't install it, Maven won't use the new pom in the child builds.